flink-libraries/flink-state-processing-api/src/main/java/org/apache/flink/state/table/SavepointConnectorOptions.java

```diff
@@ -35,6 +35,7 @@ public class SavepointConnectorOptions {
     public static final String STATE_TYPE = "state-type";
     public static final String MAP_KEY_FORMAT = "map-key-format";
     public static final String VALUE_FORMAT = "value-format";
```

Comment:
We should deprecate and rename this config to `value-class`

Reply:
Same here.